packages/components/src/mobile/utils/keycodes.native.js

```diff
@@ -0,0 +1,9 @@
+export const KEYCODES_MOBILE = {
+	13: 'Enter',
```

Comment:
I didn't use the values from `@wordpress/keycodes` just in case those get deprecated with this `keyCode` being deprecated effort.